airbyte-integrations/connectors/source-faker/source_faker/airbyte_message_with_cached_json.py

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from airbyte_cdk.models import AirbyteMessage
+
+
+class AirbyteMessageWithCachedJSON(AirbyteMessage):
+    """I a monkeypatch to AirbyteMessage which pre-renders the JSON-representation of the object upon initialization.  This allows the JSON to be calculated in the thread/process that builds the object rather than the main thread."""
```

Comment:
```suggestion
    """I a monkeypatch to AirbyteMessage which pre-renders the JSON-representation of the object upon initialization.  This allows the JSON to be calculated in the process that builds the object rather than the main thread."""
```